src/Forms/InputLabel.tsx

```diff
@@ -31,7 +31,7 @@ export interface StyledLabelTextProps {
   disabled?: boolean;
 }
 
-export interface LabelProps {
+export interface InputLabelProps {
```

Comment:
Not part of the diff, but similar to the `CheckLabel` we should probably limit the allowed values of labelPosition to `POSITION.LEFT | POSITION.TOP` here and in the `TextInput` and `Dropdown` components

Reply:
Agreed, I added the type `InputLabelPosition` in commit c4c61a3.